src/Cache/Cache.php

```diff
@@ -388,7 +442,37 @@ public static function readMany($keys, $config = 'default')
     {
         $engine = static::engine($config);
 
-        return $engine->readMany($keys);
+        return $engine->getMultiple($keys);
+    }
+
+    /**
+     * Get multiple keys from the cache.
+     *
+     * ### Usage:
+     *
+     * Reading multiple keys from the active cache configuration.
+     *
+     * ```
+     * Cache::getMultiple(['my_data_1', 'my_data_2]);
+     * ```
+     *
+     * Reading from a specific cache configuration.
+     *
+     * ```
+     * Cache::getMultiple(['my_data_1', 'my_data_2], null, 'long_term');
+     * ```
+     *
+     * @param array $keys an array of keys to fetch from the cache
+     * @param mixed $default Default value
+     * @param string $config optional name of the configuration to use. Defaults to 'default'
+     * @return array An array containing, for each of the given $keys, the cached data or false if cached data could not be
+     * retrieved.
+     */
+    public function getMultiple($keys, $default = null, $config = 'default')
```

Comment:
should we have array typehint here?

Reply:
https://github.com/php-fig/simple-cache/blob/master/src/CacheInterface.php#L58
So the typehint in docblock should be changed from `array` to `iterable`. Similarly for all `*Multiple()` methods.